HarmonyCore/FileIO/FileChannelManager.dbl

```diff
@@ -50,24 +106,210 @@ namespace Harmony.Core.FileIO
 
         endclass
 
+        private static method SetConnectionLoadCount, int
+        proc
+            data envvar, @string
+            data val, a3
+            data len, d3
+            envvar = "XFCONNECTIONLOAD"
```

Comment:
Since this is Harmony Core Specific, can you put 'HC_' in front of your env var

Reply:
Done

Reply:
sorry I didn't mean the variable declared in code, but the name of the environment variable 'HC_XFCONNECTIONLOAD'

Reply:
Ok, I'll update it.

Reply:
Done